datadog/plugins/init.sls

```diff
@@ -0,0 +1,20 @@
+{% from "datadog/map.jinja" import datadog with context %}
+
+include:
+  - ..service
+
+datadog_plugin_configuration_directory:
+  file.directory:
+    - name: /etc/dd-agent/conf.d
+
+{% for name, settings in salt.pillar.get('datadog:integrations', {}).items() %}
```

Comment:
I know this works, but is there any reason to not use the merged `datadog` context rather than the pillar data directly?